test/smoke/src/main.ts

```diff
@@ -144,7 +149,12 @@ async function setup(): Promise<void> {
     console.log("*** Preparing smoke tests setup...");
     appiumHelper.runAppium();
 
+    if (process.platform === "darwin") {
```

Comment:
Are you sure that running both simulator and emulator  at one time is a good idea? It may affect the machine performance. I think it would be better if we create entry point for macOS tests and then will run simulator when the Android tests is finished and emulator is off.

Reply:
@SounD120 we'll see. As for now, I think we can leave it as it is and in the future refactor this part.